libs/styles/lazy-styles.css

```diff
@@ -1 +1,133 @@
-/* below the fold CSS goes here */
\ No newline at end of file
+/* below the fold CSS goes here */
+
+/* article card styles, share between blocks */
+
```

Comment:
I like having a shared article card styles, but Im not sure this is the right location for it. Maybe move to its own file and include it in the various card blocks that you need so its only called when used.